extensions/mongodb-client/runtime/src/test/java/io/quarkus/mongodb/DeploymentWatchTest.java

```diff
@@ -26,5 +26,6 @@ void test() throws InterruptedException {
         collection.updateOne(eq("hello", "world"), new Document("$set", new Document().append("hello", "foo")));
         new Thread(() -> collection.watch().forEach(printBlock)).start();
         Thread.sleep(5000);
+        mongoClient.close();
```

Comment:
AFAICS, this test is not executed. And with a `Thread.sleep(5000)`, I would say it's a good choice.

Anyway, if we fix it, let's use a try with resources.

Reply:
I think this test is there only for investigation.
I can convert it to try-with-resources but would keep it disabled.
@cescoffier agree with this approach ?